src/test/java/lab4/game/BoardTest.java

```diff
@@ -0,0 +1,95 @@
+package lab4.game;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static lab4.game.Board.Status.*;
+
+
+
+
+public class BoardTest {
+
+    @Test
+    void testBoardIsEmpty() {
+        var board = new Board();
+        assertEquals(InProgress, board.getStatus());
```

Comment:
A few more tests for InProgress would be more thorough.  E.g. what about a board with just one token, or all but one spaces filled?